apps/angular-lib-dev/src/app/dropdown/dropdown.module.ts

```diff
@@ -0,0 +1,24 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+
+import { DropdownRoutingModule } from './dropdown-routing.module';
+import { DropdownComponent } from './dropdown.component';
+import { ReactiveFormsModule } from '@angular/forms'
+// eslint-disable-next-line
+import { GreenDropdownModule } from '@sebgroup/green-angular'
```

Comment:
Green seems redundant in the name. It's a DropdownModule from `green-angular`. That should be enough.

Reply:
Otherwise it seems fine